src/objects/media/media.scss

```diff
@@ -36,6 +36,16 @@
   order: 1;
 }
 
+/**
+ * Modifier: Jaunty
+ *
+ * Applies a fun rotation to the media object
+ */
+
+.o-media--jaunty .o-media__object {
+  transform: rotate(-5deg);
```

Comment:
Should we apply the inverse when the media object is reversed?

Reply:
Done!